packaging/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# This Dockerfile is based on the recommendations provided in the
```

Comment:
Can we create a common docker image for preparing CentOS7 for distribution and the CI and then just do the extra changes for CI/distribuition in a separate Dockerfile? Or have one Dockerfile create multiple images?

Reply:
So I think we can use this image as the base, the CI can be based on it and add its adjustments.